apps/web/utils/supabase/client.ts

```
@@ -3,8 +3,85 @@
+        } catch (e) {
+          // Fall through to normal client
+        }
+      }
+    }
+  }
+
+  return;
   createBrowserClient<Database>(
     process.env.NEXT_PUBLIC_SUPABASE_URL!,
     process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY!,
   );
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)